code/modules/mob/living/living_defense.dm

```diff
@@ -303,7 +303,10 @@
 		M.do_attack_animation(src)
 		visible_message("<span class='danger'>\The [M] [M.attacktext] [src]!</span>", \
 						"<span class='userdanger'>\The [M] [M.attacktext] [src]!</span>")
-		add_attack_logs(M, src, "Animal attacked")
+		if(ckey)

```

Comment:
Preferable way to do this is `add_attack_logs(M, src, "Animal atttacked", ckey ? TRUE : FALSE)`

Reply:
Changed